srcpkgs/python3-MechanicalSoup/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'python3-MechanicalSoup'
+pkgname=python3-MechanicalSoup
+_pkgname=MechanicalSoup
+version=0.11.0
+revision=1
+wrksrc=${_pkgname}-${version}
+build_style=python3-module
+hostmakedepends="python3-setuptools"
```

Comment:
is missing depends=

depends="python3 python3-requests python3-six python3-lxml python3-BeautifulSoup4"

Reply:
please cut-off at 80 columns into a new line and prefix the new line with 1 whitespace

Reply:
Done.  Looks weird to me, but just something I'll get used to. Is that something xlint could flag on?

Reply:
No, but it is pretty easy to see.

see gtk+3 dependencies as an example